### PR TITLE
hyperpose compatible

### PR DESCRIPTION
**Pull request by Gyx-One**
(1)maxpool and batchnorm dataformat debuged,support "channels_first"
(2)vgg forward fixed

### Checklist
- [x] I've tested that my changes are compatible with the latest version of Tensorflow.
- [x] I've read the [Contribution Guidelines](https://github.com/tensorlayer/tensorlayer/blob/master/CONTRIBUTING.md)
- [ ] I've updated the documentation if necessary.

### CODE REVIEW

**Comment by zsdonghao** [comment]
make a new release?
